README.md

```diff
@@ -53,3 +48,32 @@ Starting with the Yelp Reviews Extractor, we get the **Stars** and the **Free Te
 
 Also, bellow we show the use cases in B.A.C.O.N.
 ![uc](images/use_cases.png)
+
+The six main tasks of B.A.C.O.N consist in retrieve a list of restaurants based in London that have a Twitter account. With this list, we can search for tweets about them and use this data in a sentiment analysis algorithm. The process to get the Rank for London Restaurants is done after we apply the Weight Model over our Classified Tweets.
```

Comment:
Please change "consist in retrieve" to "consist in retrieving".

Please change "tweets about them" to "tweets about them (i.e., mentions),"